internal/flag/flags.go

```diff
@@ -192,6 +192,7 @@ const (
 	CASFilePath                     = "casFile"                         // CASFilePath flag
 	MonthsUntilExpiration           = "monthsUntilExpiration"           // MonthsUntilExpiration flag
 	IP                              = "ip"                              // IP flag
+	IPAddresses                     = "ipAddresses"                     // IPAddresses flag
```

Comment:
[nit] we try to keep flags singular (even when they are lists) also there's this flag already, `AccessListIP` if you feel like reusing that one

Reply:
sure

Reply:
Done